stepflow-rs/crates/stepflow-execution/src/workflow_executor.rs

```diff
@@ -37,6 +37,28 @@ pub(crate) async fn execute_workflow(
     run_id: Uuid,
     input: ValueRef,
     state_store: Arc<dyn StateStore>,
+) -> Result<FlowResult> {
+    execute_workflow_with_overrides(
```

Comment:
nit: method level comment saying we delegate to execute_workflow_with_overrides with an empty WorkflowOverrides.